hrjobcontract/tests/phpunit/CRM/Hrjobcontract/Import/Parser/ApiTest.php

```diff
@@ -265,6 +265,27 @@ public function testFTEFieldsAreSetToZeroWhenImportingOnlyMandatoryFields() {
     $this->validateHourAutoFields($contact3['id'], $expected);
   }
 
+  public function testActualHoursIsDeducedForFullTimeHoursType() {
```

Comment:
Just a comment: I think that the work "deduced" doesn't make sense here. For me, deduce is more about logical reasoning (when you deduce something based on some facts/details), which is different of what actually happens here, where you are actually copying the value from one field to others